lib/mutant/mutator/node.rb

```diff
@@ -56,10 +56,10 @@ def children
       # @return [undefined]
       #
       # rubocop:disable RedundantBlockCall - its not redundant here
-      def mutate_child(index, mutator = Mutator, &block)
+      def mutate_child(index, &block)
         block ||= TAUTOLOGY
-        child = children.at(index)
-        mutator.each(child, self) do |mutation|
+        child = children.fetch(index)
```

Comment:
Same refactoring left over as you spotted elsewhere. I was focusing on a different area when I passed this code.

I do think this can easily be a mutation. If the synthetic lvar can be inlined with its only use-side, the mutation is alive.

If its important from side effectful sequencing, mutation will be killed.